pkg/apis/policies/v1/policyserver_webhook.go

```diff
@@ -103,3 +121,31 @@ func (v *policyServerValidator) ValidateUpdate(ctx context.Context, _, obj runti
 func (v *policyServerValidator) ValidateDelete(_ context.Context, _ runtime.Object) (admission.Warnings, error) {
 	return nil, nil
 }
+
+func validateLimitsAndRequests(limits, requests corev1.ResourceList) error {
+	if requests == nil || limits == nil {
+		return nil
+	}
+
+	for limitName, limitQuantity := range limits {
+		if limitQuantity.Cmp(resource.Quantity{}) < 0 {
+			return fmt.Errorf("%s limit must be greater than or equal to 0", limitName)
+		}
+	}
+
+	for requestName, requestQuantity := range requests {
+		if requestQuantity.Cmp(resource.Quantity{}) < 0 {
+			return fmt.Errorf("%s request must be greater than or equal to 0", requestName)
+		}
```

Comment:
Does make sense to have zero quantities? Furthermore, I think this can cause issues. For instance, I'm not sure if  Kubernetes allow defining memory with `0` quantity. It looks weird. Therefore, I suggest to not allow zero quantities neither.

Reply:
Weirdly enough,`0` is a valid value for Kubernetes in this context, it just means that no limits are set for the given resource